src/tee/datatype.ts

```diff
@@ -79,17 +79,53 @@ export const isMarkdownDoc = (doc: Doc<unknown>): doc is MarkdownDoc => {
   return !!typedDoc.content && !!typedDoc.commentThreads;
 };
 
+const promptForAIChangeGroupSummary = ({
+  docBefore,
+  docAfter,
+}: {
+  docBefore: MarkdownDoc;
+  docAfter: MarkdownDoc;
+}) => {
+  return `
+Summarize the changes in this diff in a few words.
+
+Only return a few words, not a full description. No bullet points.
+
+Here are some good examples of descriptive summaries:
+
+wrote initial outline
+changed title
+small wording changes
+turned outline into prose
+lots of small edits
+total rewrite
+a few small tweaks
+reworded a paragraph
+
+## Doc before
+
+${JSON.stringify(pick(docBefore, ["content", "commentThreads"]), null, 2)}
+
+## Doc after
+
+${JSON.stringify(pick(docAfter, ["content", "commentThreads"]), null, 2)}`;
+};
+
```

Comment:
It's a bit annoying that we already have absolute positions here that we then turn into cursors, so we have to resolve them later back to absolute positions